build/index.css

```diff
@@ -0,0 +1,14 @@
+#meet {
+    padding-top: 56.25%;
+    width: 100%;
+    position: relative;
+}
+#meet iframe {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+}
+
+.achive {
+    margin-top: -0.8em;
+}
```

Comment:
Ja entferne ich dann einfach wieder weil das zusammen mit dem Tailwind script in einer anderen Datei erstellt wird